# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gnome-launch-box/compile.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- gnome-launch-box-0.2/tests/search-applications.c.vanilla	2007-03-25 12:43:57.000000000 +0200
+++ gnome-launch-box-0.2/tests/search-applications.c	2007-03-25 12:44:06.000000000 +0200
@@ -22,7 +22,7 @@
 
 #include <libgnomevfs/gnome-vfs.h>
 #define GMENU_I_KNOW_THIS_IS_UNSTABLE
-#include <gnome-menus/gmenu-tree.h>
+#include <gmenu-tree.h>
 
 static void
 applications_add_directory (GSList **applications, GMenuTreeDirectory *dir)
